docs/platforms/apple/common/index.mdx

```diff
@@ -85,9 +91,130 @@ If you prefer, you can also [set up the SDK manually](/platforms/apple/guides/io
 
 To capture all errors, initialize the SDK as soon as possible, such as in your `AppDelegate` `application:didFinishLaunchingWithOptions` method:
 
-<PlatformSection notSupported={["apple.tvos", "apple.watchos", "apple.visionos"]}>
+<PlatformSection supported={["apple.ios"]}>
+
+```swift {tabTitle:Swift} {"onboardingOptions": {"performance": "13-16", "profiling": "24-34", "session-replay": "17-22"}}
```

Comment:
iOS+macOS shared snippets before, needed to separate so I can add SR only to the iOS snippet